src/components/TodoListItem/TodoListItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './TodoListItem.module.css';
+
+const TodoListItem = ({ todo, onRemoveTodo }) => {
+  return (
+    <React.Fragment>
```

Comment:
Unless there's a specific reason to be using one, it's best to just use the shorthand fragment `<>`

Reply:
Updated.